features/merge_articles.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Merge Blog
+  In order to manage blogs
+  As an admin
+  I want to consolidate similar articles
+
+  Background:
+    I am logged into the admin panel
+
+  Scenario: Merge articles
```

Comment:
A potential future scenario to add would be to ensure that the merge functionality is not visible for non-admins